src/packages.builds

```diff
@@ -27,6 +27,9 @@
     <Project Include="$(MSBuildThisFileDirectory)..\pkg\Microsoft.NETCore.Platforms\Microsoft.NETCore.Platforms.builds">
       <AdditionalProperties>$(AdditionalProperties)</AdditionalProperties>
     </Project>
+    <Project Include="..\src\System.Console\pkg\System.Console.builds">
```

Comment:
This package doesn't build individually any longer in 2.0 so this isn't necessary.